packages/react-components/react-swatch-picker-preview/stories/SwatchPicker/SwatchPickerPopup.stories.tsx

```diff
@@ -1,6 +1,15 @@
 import * as React from 'react';
-import { makeStyles, shorthands, Button, Popover, PopoverSurface, PopoverTrigger } from '@fluentui/react-components';
+import {
+  makeStyles,
+  shorthands,
+  Button,
+  Popover,
+  PopoverSurface,
+  useRestoreFocusTarget,
+  useId,
+} from '@fluentui/react-components';
 import { SwatchPicker, SwatchPickerOnSelectEventHandler, ColorSwatch } from '@fluentui/react-swatch-picker-preview';
```

Comment:
As we are importing values and types separately from `@fluentui/react-components`, can we do the same with `@fluentui/react-swatch-picker-preview`?

Reply:
we don't have it in `react-components` yet